website/www/site/content/en/documentation/transforms/python/elementwise/pardo.md

```diff
@@ -76,10 +76,12 @@ starts and finishes with `start_bundle` and `finish_bundle`.
   multiple instances of a given `DoFn` subclass may be created (e.g., due to parallelization, or due to garbage collection after a period
   of disuse).
   This is a good place to connect to database instances, open network connections or other resources.
+  See also `DoFn.SetupContextParam` for a way to accomplish this via context managers.
```

Comment:
1. How do we make this documentation content properly discoverable without having users go to search for a docstring on internal class in Beam codebase in a GH repo? Should DoFn.BundeContextParam copy the docstring so that it is visible in https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.core.html#apache_beam.transforms.core.DoFn.process, or ?

2. Let's update the `The full set of parameters is:` section in  
https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.core.html#apache_beam.transforms.core.DoFn.process